test/UnitTests/GitHub.VisualStudio/Services/UsageTrackerTests.cs

```diff
@@ -8,209 +7,268 @@
 using GitHub.Settings;
 using NSubstitute;
 using NUnit.Framework;
+using System.Reflection;
+using System.Linq;
 
-
-namespace UnitTests.GitHub.VisualStudio.Services
+namespace MetricsTests
```

Comment:
I don't have a problem with this, but for reference many other unit tests have no namespace, e.g. https://github.com/github/VisualStudio/blob/master/test/UnitTests/GitHub.Exports/GitServiceTests.cs. However we're not consistent about this it seems.